modules/robin/src/test/ts/browser/LanguageOverrideTest.ts

```diff
@@ -0,0 +1,72 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { DomUniverse, Universe } from '@ephox/boss';
+import { Arr, Fun } from '@ephox/katamari';
+import { SugarElement, Traverse } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import { TextZones, ZoneViewports } from 'ephox/robin/api/Main';
+
+// Strip out the bits of an actual Zone that aren't compatible with assert.deepEqual
+interface AssertableZone {
+  readonly words: string[];
+  readonly lang: string;
+}
+
+describe('browser.robin.LanguageOverrideTest', () => {
+  const top = SugarElement.fromHtml('<div>' +
+    '<p lang="en">Hello <span>world</span></p>' +
+    '<p>This is <span lang="pt">multi-lingual content</span></p>' +
+    '</div>'
+  );
+
+  const walk = (universe: Universe<SugarElement, Document>): Array<AssertableZone> => {
```

Comment:
```suggestion
  const walk = (universe: Universe<SugarElement, Document>): AssertableZone[] => {
```
`Array` should only be used for complex types (I miss the old tslint rule that could enforce this 😢)